source/setup.h

```diff
@@ -2,7 +2,61 @@
 #define SETUP_H
 
 #include <wafel/types.h>
+#include <wafel/ios/svc.h>
```

Comment:
better have that in it's own header compaining the c file for the Sysprod functions